db/migrate/20170203113625_remove_organisation_reference_from_content_items.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveOrganisationReferenceFromContentItems < ActiveRecord::Migration[5.0]
+  def change
+    remove_reference :content_items, :organisation, foreign_key: true
```

Comment:
I would prefer not to make developers reload all the data unless it is needed, because:

- It takes a while to load all the data from scratch
- We would also need to do it in Heroku

If there is a way to prevent this from happening (I have checked the code above and it is working), do you have any particular reason for not addressing this comment?